main.go

```diff
@@ -5,15 +5,18 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
+	"path"
 
 	"github.com/dequelabs/nav-json-validator/navjson"
 )
 
 var silent *bool
 var file *string
+var skipFileCheck *bool
 
 func init() {
 	silent = flag.Bool("silent", false, "Silence output")
+	skipFileCheck = flag.Bool("skip-file-check", false, "Skip file existence check")
```

Comment:
In order to avoid a breaking change, we could default this to `true`. Thoughts?

Reply:
Yup, that is a nice toggle/ option to have.

I am currently validating docs/ nav.json in one of the repositories directly from this branch of the validator to see how things go.